### PR TITLE
Inverted logic in Controllable

### PR DESCRIPTION
**Pull request by Jonarod**
If Action is NOT valid, then throw error.

### CODE REVIEW

**Comment by alexisvisco** [comment]
Thank you for your pull request!

File: go.mod [hunk 1]
```diff
@@ -1,3 +1,3 @@
-module github.com/expectedsh/go-sonic
+module github.com/Jonarod/go-sonic
```
Comment (timurgarif) [inline go.mod:1]: @alexisvisco @Jonarod 
I'm sorry, but when I run
`go get -u "github.com/expectedsh/go-sonic/sonic"`
I receive
> module declares its path as: github.com/Jonarod/go-sonic
>                 but was required as: github.com/expectedsh/go-sonic

And I guess, this file merge is the reason. Please revert it back.

Comment (alexisvisco) [reply go.mod:1]: Ho well ..

**Comment by alexisvisco** [comment]
You can now update your module :) Sorry

**Comment by timurgarif** [comment]
Thank you. It's all right :)

**Pull request merged**